### PR TITLE
Milestone three NewsWeek site clone

### PR DESCRIPTION
**Pull request by ShoiraTa**
#Milestone three

 Built the following sections: 

- Newspaper main sections
 - US
 - World
 - Business
 - Tech & Science
 - Culture
 - Sports
 - Expert forum
 
- Subscribe / Newsletter

Built mobile version responsive; sections are following same as in original web page:

image

### CODE REVIEW

**Comment by mimipeshy** [comment]
### STATUS: CHANGES REQUESTED 🔴 ♻️ 

Hello there @ShoiraTa and @tanzila-abedin 👋🏾 👋🏾 , great job so far on this milestone. I Am impressed by your work it looks like the original page. 
✔️  Great job on the PR description it's well written
However, before approval, let's make it even better

#### In the magazine and Editor's pick

* [x] The title has the Newsweek logo

That's all :smile: kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, or just comment on this PR and tag my name and I will be glad to help_
Regards
Peris Ndanu

**Comment by tanzila-abedin** [comment]
Hello @mimipeshy 

We have added the news week logo to the magazine and editor's pick sections.

Thank you

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, congratulations on adding the logo to the title of the "In the magazine" and the "Editors Pick" Awesome :muscle: However, I came across the social handles of the authors in the README file and I realize none of the social media handles actual worked. We are required to use our social media handles so that any person being it recruiter/Interviewer can get to know more about the developer. As it stands now, we need to fix it before proceeding. Let us kindly fix it :smile: 
Also, the steps in cloning and running the project were not explicitly shown

[Optional]
- Let us improve the use of HTML semantics like the `<section>, <article>` etc so we can actually describe the application

Once this is fixed, kindly submit another code review request :slightly_smiling_face: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by tanzila-abedin** [comment]
Hello

The READ.ME  file was updated with active social media links.

Thank you

File: README.md
Comment (christianotieno) [inline README.md:47]: ## \[Optional\]: Improve the LinkedIn links 
- [x]  Great job fixing the social links. I would advise if possible, we remove the Author tags and just have only our names in these sections since the title is already there in the heading of this section in this `README. and it would be much neater as well.  What are your thoughts on this? Thank you!

```markdown
👤 **Tanzila**

 GitHub: [@tanzila-abedin](https://github.com/tanzila-abedin)
- LinkedIn: [Tanzila Abedin](https://www.linkedin.com/in/tanzila-abedin-331440b2/)

👤 **Shoira**

- GitHub: [@shoirata](https://github.com/ShoiraTa)
- LinkedIn: [Shoira Tashpulatova](https://www.linkedin.com/in/shoira-tashpulatova-bab4a7122)

```

**Review by christianotieno — APPROVED** [review]
### Milestone Approved 🍾 
Hello @ShoiraTa & @tanzila-abedin 👋 
##
This milestone has now met all the requirements needed for approval. 👏 
Congratulations on the hard work done with the changes and overall, on this milestone. Good job :+1:
I will go ahead and mark this as complete. Before we merge, I, however, do have a small suggestion for us down below 👇 that you might want to consider. It is very much optional, though. Good luck with the next tasks 🥇 
##
### Happy Coding 💻 
#### Christian Otieno (CR)
**| [Twitter](https://twitter.com/iamchrisotieno) | [Github](https://github.com/christianotieno) | [LinkedIn](https://www.linkedin.com/in/christianotieno/) |**

File: README.md
Comment (tanzila-abedin) [reply README.md:47]: Thank you  @christianotieno .  👍🏽 
I have removed those author tags as you suggested.

**Pull request merged**
